packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/general-information.html

```diff
@@ -373,28 +401,70 @@
+                    </oui-tile-description>
+                </oui-tile-definition>
+                <oui-tile-definition
+                    data-term="{{:: 'pci_databases_general_information_new_fork_label' | translate }}"
+                    data-ng-if="$ctrl.isFeatureActivated('forkFromGeneralInformation')"
+                >
+                    <oui-tile-description>
+                        <button
+                            class="oui-tile__button oui-button oui-button_ghost oui-button_block"
+                            data-ng-click="$ctrl.goToFork()"
+                        >
+                            <span
+                                data-translate="pci_databases_general_information_new_fork_link"
+                            ></span>
+                            <span
+                                class="oui-icon oui-icon-arrow-right ml-1"
+                                aria-hidden="true"
+                            ></span>
+                        </button>
+                    </oui-tile-description>
+                </oui-tile-definition>
                 <oui-tile-definition
                     data-term="{{:: 'pci_databases_general_information_allowed_ips_label' | translate }}"
                 >
                     <oui-tile-description>
                         <span
-                            class="oui-badge"
-                            data-ng-class="{
-                                'oui-badge_success': ($ctrl.allowedIps.length >= 1),
-                                'oui-badge_warning': ($ctrl.allowedIps.length < 1),
-                            }"
+                            class="oui-badge oui-badge_warning"
+                            data-ng-if="$ctrl.allowedIps.length === 0"
                         >
                             <span
-                                data-ng-if="$ctrl.allowedIps.length === 0"
                                 data-translate="pci_database_common_none"
                             ></span>
-                            <p
-                                class="mb-0"
-                                data-ng-if="$ctrl.allowedIps.length !== 0 && $index < 3"
-                                data-ng-repeat="allowedIp in $ctrl.allowedIps track by allowedIp.ip"
-                                data-ng-bind="allowedIp.ip"
-                            ></p>
                         </span>
+
+                        <p
+                            data-ng-if="$ctrl.allowedIps.length !== 0 && $index < 3"
+                            data-ng-repeat="allowedIp in $ctrl.allowedIps track by allowedIp.ip"
+                        >
+                            <span
+                                class="oui-badge oui-badge_success"
+                                data-ng-bind="allowedIp.ip"
+                            ></span>
+                            <span
+                                data-ng-if="$ctrl.allowedIps.length > 3 && $index === 2"
```

Comment:
Move the number `3` to a constant and use that constant-1 for 2

Reply:
Done here: https://github.com/ovh/manager/pull/5583